Troubleshooting: Component Content is not Scrollable (#5442)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/45 Component Content is not Scrollable/15 Component Height is 100% But It Does Not Fit Into the Page.md

> This situation usually occurs when none of the component parent elements has a fixed height.

Reviewer: ```suggestion
This situation usually occurs when none of parent elements has a fixed height.
```

> If the height of the component's parent is relative, the parent ancestor should have a fixed height. If there are multiple ancestors with relative heights, at least one should have a fixed height.

Reviewer: It feels like we can just say this. I believe it expresses everything. If at least one parent has relative height, at least one of it's parents should set a fixed height. Right? 

```suggestion
If a parent element sets a relative height, one of its parents should set a fixed height. 
```

> This situation usually occurs when none of parent elements has a fixed height. For example, on the image below DataGrid does not fit the page.

Reviewer: ```suggestion
This situation usually occurs when none of parent elements has a fixed height. For example, on the image below DataGrid does not fit the page. Page scrollbar appears. 
```
